tests/e2e/tests/regression/workbench/redis-stack-commands.e2e.ts

```diff
@@ -0,0 +1,74 @@
+import { acceptLicenseTermsAndAddDatabase, deleteDatabase } from '../../../helpers/database';
+import { WorkbenchPage } from '../../../pageObjects/workbench-page';
+import { MyRedisDatabasePage } from '../../../pageObjects';
+import {
+    commonUrl,
+    ossStandaloneConfig
+} from '../../../helpers/conf';
+import { env, rte } from '../../../helpers/constants';
+import { t } from 'testcafe';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const workbenchPage = new WorkbenchPage();
+const keyNameGraph = 'bikes_graph';
+
+fixture `Redis Stack command in Workbench`
+    .meta({type: 'regression'})
+    .page(commonUrl)
+    .beforeEach(async t => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        await t.click(myRedisDatabasePage.workbenchButton);
+    })
+    .afterEach(async() => {
+        //Drop key and database
+        await t.switchToMainWindow();
+        await workbenchPage.sendCommandInWorkbench(`GRAPH.DELETE ${keyNameGraph}`);
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+//skipped due the inaccessibility of the iframe
+test.skip
+    .meta({ rte: rte.standalone })
+    ('Verify that user can switches between Graph and Text for GRAPH command and see results corresponding to their views', async t => {
+        //Send Graph command
+        await t.click(workbenchPage.redisStackTutorialsButton);
+        await t.click(workbenchPage.workingWithGraphLink);
+        await t.click(workbenchPage.createGraphBikeButton);
+        await t.click(workbenchPage.submitCommandButton);
+        //Switch to Text view and check result
+        await workbenchPage.selectViewTypeText();
+        await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssQueryTextResult).exists).ok('The text view is switched for GRAPH command');
+        //Switch to Graph view and check result
+        await workbenchPage.selectViewTypeGraph();
+        await t.switchToIframe(workbenchPage.iframe);
+        await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.queryGraphContainer).exists).ok('The Graph view is switched for GRAPH command');
+    });
+test
+    .meta({ rte: rte.standalone })
+    ('Verify that user can see "No data to visualize" message for Graph command', async t => {
+        //Send Graph command
+        await t.click(workbenchPage.redisStackTutorialsButton);
+        await t.click(workbenchPage.workingWithGraphLink);
+        await t.click(workbenchPage.preselectModelBikeSalesButton);
+        await t.click(workbenchPage.submitCommandButton);
```

Comment:
Make a function within the file if this action is repeating.

Reply:
this steps are bot repeating